onadata/libs/utils/logger_tools.py

```diff
@@ -403,14 +404,16 @@ def safe_create_instance(username, xml_file, media_files, uuid, request):
         error = OpenRosaResponseBadRequest(
             _(u"Unable to submit because there are multiple forms with"
               u" this formID."))
+    except DataError as e:
+        error = OpenRosaResponseBadRequest(
+            'Data Incompatible with the database provided')
```

Comment:
According to the exceptions, it seems we had a calculate field that returned an Infinity result. I am wondering whether rejecting such submissions is the right way to go about it. Can we be clear in our message of the issue so that an enumerator can know what to check and change on their end? Or should we accept the result as is and replace the Infinity with something that can be stored in a JSON field?
It would be ideal to see if we can actually replicate the issue to determine if the issue is in the data collection tools Enketo or ODK Collect and what could be potentially be solved on that end.